tests/marketplace/marketplace_four_integration_test.py

```diff
@@ -62,3 +63,34 @@ def test_should_upload_file(oauth_api):
     request.purpose = 'identification'
     response = oauth_api.marketplace.upload_file(request)
     assert_response(response, 'id', '_links')
+
+
+def test_should_initiate_transfer_of_funds(oauth_api):
+    transfer_source = TransferSource()
+    transfer_source.id = 'ent_kidtcgc3ge5unf4a5i6enhnr5m'
+    transfer_source.amount = 100
+
+    transfer_destination = TransferDestination()
+    transfer_destination.id = 'ent_w4jelhppmfiufdnatam37wrfc4'
+
+    transfer_request = CreateTransferRequest()
+    transfer_request.transfer_type = TransferType.COMMISSION
+    transfer_request.source = transfer_source
+    transfer_request.destination = transfer_destination
+
+    response = oauth_api.marketplace.initiate_transfer_of_funds(transfer_request)
+    assert_response(response, 'id', 'status')
+    assert 'pending' == response.status
+
+
+def test_should_retrieve_entity_balances(oauth_api):
+    query = BalancesQuery()
+    query.query = 'currency'.join(Currency.GBP)
+
+    response = oauth_api.marketplace.retrieve_entity_balances('ent_kidtcgc3ge5unf4a5i6enhnr5m', BalancesQuery)
```

Comment:
I believe this is a creating a new `BalancesQuery` instead of using the created one `query`.

Reply:
Just fixed it.